web/src/components/profile/ProfileErrorAlert.tsx

```diff
@@ -0,0 +1,61 @@
+import { Alert } from "@/components/njwds-extended/Alert";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { getProfileConfig } from "@/lib/domain-logic/getProfileConfig";
+import { ProfileContentField } from "@/lib/types/types";
+import { templateEval } from "@/lib/utils/helpers";
+import { ReactElement } from "react";
+
+interface Props {
+  fieldErrors: string[];
+}
+
+export const ProfileErrorAlert = (props: Props): ReactElement => {
+  const { Config } = useConfig();
+
+  const displayProfileErrorAlert = (): boolean => props.fieldErrors.length > 0;
```

Comment:
I copied and pasted this comment in two places

We had previously talked about how we should separate the UI logic from the conditional logic on whether it should display. I think we agreed it should be defined at the parent level. Therefore, `fieldErrors` does not need to be a prop.